README.md

```diff
@@ -55,7 +55,7 @@ To get started with a local [kind](https://kind.sigs.k8s.io/) Kubernetes cluster
    To access it from your local machine, use the following port-forward command:
 
    ```bash
-   kubectl port-forward svc/orchestration-api 8000:8010 -n orchestration-api
+   kubectl port-forward svc/orchestration-api 8000:8000 -n orchestration-api
```

Comment:
Make it 8010:8000

generally 8000 might be used in local machines and we can clearly show which one is local and which one remote.